src/pretalx/locale/es_MX/LC_MESSAGES/django.po

```diff
@@ -2841,10 +2860,6 @@ msgstr ""
 msgid "The session's room"
 msgstr "La imagen de encabezado de esta charla"
 
-#: pretalx/mail/context.py:247
-msgid "Jane Doe"
-msgstr "Fulanita de Tal"
-
```

Comment:
Isn't this relevant for https://github.com/fossasia/eventyay-talk/blob/297c53ef94c98c28ce195fe16a9a23f5e974b2b7/src/pretalx/mail/context.py#L247?

Reply:
The same answer applies here.